vcd/resource_vcd_nsxt_alb_virtual_service.go

```diff
@@ -0,0 +1,346 @@
+package vcd
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceVcdAlbVirtualService() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceVcdAlbVirtualServiceCreate,
+		ReadContext:   resourceVcdAlbVirtualServiceRead,
+		UpdateContext: resourceVcdAlbVirtualServiceUpdate,
+		DeleteContext: resourceVcdAlbVirtualServiceDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceVcdAlbVirtualServiceImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"org": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Description: "The name of organization to use, optional if defined at provider " +
+					"level. Useful when connected as sysadmin working across different organizations",
+			},
+			"vdc": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The name of VDC to use, optional if defined at provider level",
+			},
+			"edge_gateway_id": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Edge gateway ID in which ALB Pool should be created",
+			},
+			"name": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Name of ALB Virtual Service",
+			},
+			"description": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Description of ALB Virtual Service",
+			},
+			"pool_id": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Edge gateway ID in which ALB Pool should be created",
+			},
+			"service_engine_group_id": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Edge gateway ID in which ALB Pool should be created",
```

Comment:
Wrong description :)

Reply:
Fixed this and a few more erroneous descriptions. Hopefully it should be good for all.